core/src/main/java/com/linecorp/armeria/common/util/EventLoopThreadFactory.java

```diff
@@ -37,7 +37,13 @@
      * Creates a new factory that creates a non-daemon and normal-priority thread.
      *
      * @param threadNamePrefix the prefix of the names of the threads created by this factory.
+     *
+     * @deprecated Use {@linkplain ThreadFactories#newEventLoopThreadFactory(String, boolean)
+     *             ThreadFactories#newEventLoopThreadFactory(String, false)} or
```

Comment:
I'm not sure this line break is appropriate. PTAL :bowing_man:

Reply:
I think you need to use `@link` because `@linkplain` will produce a non-monospace text.

Reply:
Confirmed the the page is rendered correctly.